pkg/JuliaInterface/src/JuliaInterface.c

```diff
@@ -57,8 +61,10 @@ Obj Func_JULIAINTERFACE_INTERNAL_INIT(Obj self)
     jl_module_t * gap_module = get_module("__JULIAGAPMODULE");
     JULIA_GAPFFE_type =
         (jl_datatype_t *)jl_get_global(gap_module, jl_symbol("FFE"));
-    if (!JULIA_GAPFFE_type)
+    if (!JULIA_GAPFFE_type) {
+        BEGIN_GAP_SYNC();
         ErrorMayQuit("Could not locate the GAP.FFE datatype", 0, 0);
+    }
```

Comment:
Huh... I wind this really irritating to read, and I think I'd prefer to have the `END_GAP_SYNC` in there. Or have a `ErrorWithSync` helper function which contains the call to `BEGIN_GAP_SYNC`, and then call that here and in similar places (I think it's fine to always use `ErrorMayQuit`, so no need for multiple such helpers).

Reply:
Right now, I've simply put the `END_GAP_SYNC()` back in, even though the code is never reached. It still serves as documentation.